runtime/doc/insert.txt

```diff
@@ -1630,28 +1632,29 @@ market. Theoretically standards are created by W3C organisation
 		-	+	-    -		     ~
 
 Regardless from state of implementation in browsers but if element is defined
-in standards, completion plugin will place element in suggestion list. When
+in standards, completion plugin will place element in suggestion list.  When
 both major engines implemented element, even if this is not in standards it
-will be suggested. All other elements are not placed in suggestion list.
+will be suggested.  All other elements are not placed in suggestion list.
```

Comment:
This whole section does no longer make sense, I to be honest, I don't understand what that table is supposed to be telling us. How about this instead:
```
The DOM standard is created by the W3C (http://www.w3c.org) and the WHATWG
(https://whatwg.org) organizations, though browser implementations may vary.

Regardless of the implementation state in browsers:
  - If an element is defined in the standard, the completion plugin will
    suggest it
  - If a feature is supported in Firefox and Internet Explorer, even if
    it is not yet standardized, it will also be suggested
  - All other elements are not placed into the suggestion list
```
Nevermind, I finally understood the table 🙈